src/coverlet.console/Program.cs

```diff
@@ -135,20 +137,47 @@ static int Main(string[] args)
                 }
 
                 var thresholdTypeFlags = ThresholdTypeFlags.None;
-
+                var thresholdTypeFlagQueue = new Queue<ThresholdTypeFlags>();
                 foreach (var thresholdType in dThresholdTypes)
                 {
                     if (thresholdType.Equals("line", StringComparison.OrdinalIgnoreCase))
                     {
                         thresholdTypeFlags |= ThresholdTypeFlags.Line;
+                        thresholdTypeFlagQueue.Enqueue(ThresholdTypeFlags.Line);
                     }
                     else if (thresholdType.Equals("branch", StringComparison.OrdinalIgnoreCase))
                     {
                         thresholdTypeFlags |= ThresholdTypeFlags.Branch;
+                        thresholdTypeFlagQueue.Enqueue(ThresholdTypeFlags.Branch);
                     }
                     else if (thresholdType.Equals("method", StringComparison.OrdinalIgnoreCase))
                     {
                         thresholdTypeFlags |= ThresholdTypeFlags.Method;
+                        thresholdTypeFlagQueue.Enqueue(ThresholdTypeFlags.Method);
+                    }
+                }
+
+                ThresholdTypeFlagValues thresholdTypeFlagValues = new ThresholdTypeFlagValues();
+                if (threshold.HasValue() && threshold.Value().Contains(','))
+                {
+                    var thresholdValues = threshold.Value().Split(',').Select(t => t.Trim());
```

Comment:
we should avoid empty entries use overload `StringSplitOptions.RemoveEmptyEntries` https://docs.microsoft.com/en-us/dotnet/api/system.string.split?view=netframework-4.8#System_String_Split_System_String___System_StringSplitOptions_